uber/site_sections/preregistration.py

```diff
@@ -181,7 +181,7 @@ def form(self, session, message='', edit_id=None, **params):
             }
 
         if attendee.is_dealer and not c.DEALER_REG_OPEN:
-            return render('static_views/dealer_reg_closed.html') if c.AFTER_DEALER_REG_SHUTDOWN \
+            return render('static_views/dealer_reg_closed.html') if c.AFTER_DEALER_REG_START \
```

Comment:
Wait, this seems like it was right before. Am I reading this wrong?

Reply:
This was fine before, but when we implement auto-close it'll cause weird behavior: if we reach the app limit and we close applications, but it's not past the `dealer_reg_shutdown` date, it would tell people that dealer reg isn't open yet!

Reply:
Ah! Makes sense.